modules/environment/login/nix-on-droid.nix.default

```diff
@@ -28,15 +28,21 @@
   environment.etcBackupExtension = ".bak";
 
   # Read the changelog before changing this value
-  system.stateVersion = "20.03";
+  system.stateVersion = "20.09";
 
   # After installing home-manager channel like
-  #   nix-channel --add https://github.com/rycee/home-manager/archive/release-20.03.tar.gz home-manager
+  #   nix-channel --add https://github.com/rycee/home-manager/archive/release-20.09.tar.gz home-manager
   #   nix-channel --update
   # you can configure home-manager in here like
   #home-manager.config =
   #  { pkgs, ... }:
   #  {
+  #    # Read the changelog before changing this value
+  #    home.stateVersion = "20.09";
+  #
+  #    # Use the same overlays as the system packages
+  #    nixpkgs.overlays = config.nixpkgs.overlays;
```

Comment:
I think, I would prefer setting `home-manager.useGlobalPkgs = true;` which syncs the complete nixpkgs instance.

Reply:
OK, I've dropped that one commit then (I don't like useGlobalPkgs enough to recommend it to everyone, too magical to my taste).